bitrise.yml

```diff
@@ -0,0 +1,109 @@
+format_version: 5
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - BITRISE_STEP_VERSION: "0.9.0"
+
+  - PUBLIC_INSTALL_PAGE_URL: $PUBLIC_INSTALL_PAGE_URL
+  - QR_CODE_SIZE: $QR_CODE_SIZE
```

Comment:
use this syntax only for secrets! (by syntax i mean defining app envs for highlight required secrets)